### PR TITLE
Update nl_divergence_tolerance test to work with PETSc 3.11.4

### PR DESCRIPTION
**Pull request by frombs**
Update nl_divergence_tolerance test to work with PETSc 3.11.4, closes issue #14598

### CODE REVIEW

**Comment by frombs** [comment]
@fdkong, this is ready for review.

**Comment by permcody** [comment]
Ping @fdkong for review. @frombs - I'm a little confused about this test. Does it require that the problem fails to converge to properly test? I admit I haven't really dug into it.

**Comment by fdkong** [comment]
@permcody @frombs  I will re-investigate  the test.  I would like to make sure the test is robust

**Comment by frombs** [comment]
@permcody, that is correct...in order to test the nl_divergence_tolerance, I had to develop a simple test that would diverge quickly. I did that by poorly scaling the variables.  It appears that either changes to PETSc or something in the MOOSE framework improved the convergence so I had to scale the variables further to get it to diverge again.

**Comment by permcody** [comment]
Ok - good. @fdkong - Can you give Brad a few suggestions on how to test this? I think that the current strategy is always going to be touchy.

File: test/tests/executioners/nl_divergence_tolerance/nl_divergence_tolerance.i [hunk 1]
```diff
@@ -50,8 +50,8 @@
   nl_max_its = 20
   nl_div_tol = 1.0e+4
```
Comment (fdkong) [inline test/tests/executioners/nl_divergence_tolerance/nl_divergence_tolerance.i]: @frombs `make nl_div_tol = 1.0e+4 small ` will have the test more robust. Change to `nl_div_tol=10`

Here we just want to make sure nobody delete `nl_div_tol`

**Review by fdkong — CHANGES_REQUESTED** [review]

**Comment by frombs** [comment]
The `peridynamics/test:auxkernels.stretch_NOSPD` test is timing out.

**Comment by permcody** [comment]
That test was reduced in size yesterday - it's merged into next. Don't worry about that.

**Comment by permcody** [comment]
@fdkong - is this what you wanted?

**Review by fdkong — APPROVED** [review]

**Pull request merged**